engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/validation/ProcessMessageStartEventMessageNameValidator.java

```diff
@@ -57,6 +57,13 @@ private void validateMessageName(
       return;
     }
     final String nameExpression = message.getName();
+    if (nameExpression == null) {
+      resultCollector.addError(
+          0,
+          "Message name/expression must not be null. It is null in "
+              + messageEventDefinition.getId());
```

Comment:
:thought_balloon: This change might result in an unexpected rejection message. 

The message name should be already checked by the [MessageValidator](https://github.com/camunda/zeebe/blob/f339fc92ed93ede99f040c9b4c602b1940bfde6a/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/MessageValidator.java#L59-L64). I assume that the validator already adds the failure message. We see the NPE because the validator chain is not interrupted. 

As a result, we might see now two error messages that mention that the message name is null. 

Please do a manual test and verify the rejection message.

Reply:
You were correct